src/entropy_rng.rs

```diff
@@ -55,6 +55,12 @@ impl EntropyRng {
     }
 }
 
+impl Default for EntropyRng {
```

Comment:
Are we sure we want this implementation? Seems ok to me.

Reply:
Seems okay and it's available in `core`; the only real reason to use it is generic programming. I don't see any reason not to add it then, though I prefer using `EntropyRng::new()` myself.